lib/src/main/java/com/orange/ods/compose/component/control/OdsSlider.kt

```diff
@@ -193,7 +187,6 @@ fun OdsSliderLockups(
                     .padding(bottom = sideIconBottomPadding),
                 painter = painter,
                 contentDescription = leftIconContentDescription,
-                tint = OdsTheme.colors.onSurface
```

Comment:
There are other similar lines in the code, should we also remove them now that the Material theme method is called?

Reply:
Yes, you're right. I think I forgot to remove some things...

Reply:
Actually, when I remove these lines, the icons are not really white in dark mode. So, I think I have to specify the tint for icons.